### PR TITLE
media-sound/gmusicbrowser: bump to newest git revision (2016-05-30)

### PR DESCRIPTION
**Pull request by bor**
the bump contains a lot of fixes, especially perl 5.24 support
use git rev bump because there no new version a long time from upstream

### CODE REVIEW

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (mgorny) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: Upstream seems to use the following version:

     VERSIONSTRING => '1.1.15.2',

Maybe you want to account for that .2 somehow? It seems to have been added in https://github.com/squentin/gmusicbrowser/commit/85ec2784365edfcf435a494119e6a4ba3791eb64.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+EAPI=6
+
+GIT_COMMIT="853840eb9dad0b59ad2dac5d303f5929b2f09f21"
+
+inherit eutils fdo-mime gnome2-utils
+
+DESCRIPTION="An open-source jukebox for large collections of mp3/ogg/flac files"
+HOMEPAGE="http://gmusicbrowser.org/"
+SRC_URI="https://github.com/squentin/${PN}/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```
Comment (mgorny) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: Minor suggestion: I prefer if the base part of the URI is good for copy-pasting, i.e. `https://github.com/squentin/gmusicbrowser` is more convenient.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+pkg_preinst() {
+	gnome2_icon_savelist
+}
+
+pkg_postinst() {
+	fdo-mime_desktop_database_update
+	gnome2_icon_cache_update
+
+	elog "Gmusicbrowser supports gstreamer, mplayer, mpv and mpg123/ogg123..."
```
Comment (mgorny) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: What is the purpose of this printing if you enforce the dependencies using USE flags anyway?

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild (hunk 1 above)
Comment (bor) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: Neither 1.1.15.1 nor 1.1.15.2 was not released (yet?).
So I'm not sure is it good to use it.
Anyway it is a last commit in repo after these versions.
Do you think `1.1.15.2_p20160530` is a good version string in this case?

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild (hunk 2 above)
Comment (bor) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: ok, I will update.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild (hunk 3 above)
Comment (bor) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: Mostly by historic reason, before USE flag's & auto-deps was introduced.
Now it is just informative message.
I think we can remove it. I'll do.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild (hunk 1 above)
Comment (mgorny) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: Yes, that was my idea. However, it's your choice.

Comment (bor) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: Thank you for the answer.
I prefer to use '1.1.15_p20160530' one.

**Comment by bor** [comment]
@mgorny please review it again.
I've applied all your comments (except version one).
Also sync release & git version as well, so now they are equal.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+		DESTDIR="${D}" \
+		iconsdir="${D%/}/usr/share/icons/hicolor" \
+		install
+
+	use doc && local HTML_DOCS=( layout_doc.html )
+	einstalldocs
+}
+
+pkg_preinst() {
+	gnome2_icon_savelist
```
Comment (mgorny) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: Didn't I mention this is no longer needed? I do that a lot lately ;-). In any case, I'll fix it while merging.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild [hunk 5]
```diff
@@ -61,13 +67,6 @@ pkg_preinst() {
 pkg_postinst() {
 	fdo-mime_desktop_database_update
 	gnome2_icon_cache_update
-
```
Comment (mgorny) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild]: Now that's a bit confusing commit set. I'll move this removal to the previous commit to make it a little less confusing.

**Comment by bor** [comment]
Ok.
Thank you.

**Pull request closed without merging**